test/exprtest/main.cpp

```diff
@@ -25,11 +25,45 @@
 
 #include <gtest/gtest.h>
 
+#include "blas_test.hpp"
 #include "blas_test_macros.hpp"
+#include "clara.hpp"
+
+struct Args args{};
+
+namespace {
+/**
+ * @fn parse_args
+ * @brief Returns a structure containing the data extracted from the
+ * command-line arguments.
+ */
+void parse_args(int argc, char** argv) {
```

Comment:
If you need this, this can be moved to a common folder so no need for duplication

Reply:
I created a blas_test::utils namespace in blas_test.hpp. It houses both "parse_args" and "fill_random". Without the namespace I was getting duplicate function declaration errors. Because of this change, all the unittest files had to be updated to use "blas_test::utils::fill_random".